components/src/core/components/CardTable/Cell/Default.vue

```diff
@@ -1,57 +1,28 @@
 <template>
-  <div v-if="showHeader" class="oxd-table-card-cell">
-    <div class="header">
-      {{ header.title }}
+  <div class="oxd-table-card-cell">
+    <div class="header" v-if="showHeader">
+      <slot name="header" :header="header" :cell="item" :row="rowItem"></slot>
+      <template v-if="!$slots['header']">
+        {{ header.title }}
+      </template>
     </div>
-    <div class="data">
+    <slot name="body" :header="header" :cell="item" :row="rowItem"></slot>
+    <div class="data" v-if="!$slots['body']">
       {{ item }}
     </div>
   </div>
-  <template v-else>
-    <a
-      v-if="rowItem.link && clickableCell"
```

Comment:
There was a requirement to support right-click behavior for cells if the cell type is a link. Is that behavior preserved?

Reply:
Seems not an issue, since founding a related new cell type below.